### ISSUE TITLE
`protoc > 3.19` support

### ISSUE THREAD

**Issue by caniko** [so1n/protobuf_to_pydantic#2]
**Is your feature request related to a problem? Please describe.**
I got an error when using the plugin, I was on latest `protoc` at the time: 
```shell
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
```

**Describe the solution you'd like**
`protoc > 3.19` support

**Describe alternatives you've considered**
Downgrade.

**Comment by so1n** [comment]
Is the error caused by `protoc` or by some other circumstances? Please attach details(For example, the command to execute `protoc` and the protobuf file, or code)

**Comment by so1n** [comment]
> **Is your feature request related to a problem? Please describe.** I got an error when using the plugin, I was on latest `protoc` at the time:
> 
> ```shell
> TypeError: Descriptors cannot not be created directly.
> If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
> If you cannot immediately regenerate your protos, some other possible workarounds are:
>  1. Downgrade the protobuf package to 3.20.x or lower.
>  2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
> ```
> 
> **Describe the solution you'd like** `protoc > 3.19` support
> 
> **Describe alternatives you've considered** Downgrade.
Is the error caused by protoc or by some other circumstances? Please attach details(For example, the command to execute protoc and the protobuf file, or code)

**Comment by pppwaw** [comment]
> > **Is your feature request related to a problem? Please describe.** I got an error when using the plugin, I was on latest `protoc` at the time:
> > ```shell
> > TypeError: Descriptors cannot not be created directly.
> > If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
> > If you cannot immediately regenerate your protos, some other possible workarounds are:
> >  1. Downgrade the protobuf package to 3.20.x or lower.
> >  2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > **Describe the solution you'd like** `protoc > 3.19` support
> > **Describe alternatives you've considered** Downgrade.
> 
> Is the error caused by protoc or by some other circumstances? Please attach details(For example, the command to execute protoc and the protobuf file, or code)

>  Traceback (most recent call last):
>   File "/usr/local/bin/protoc-gen-protobuf-to-pydantic", line 5, in <module>
>     from protobuf_to_pydantic.plugin.main import main
>   File "/usr/local/lib/python3.9/dist-packages/protobuf_to_pydantic/plugin/main.py", line 19, in <module>
>     from protobuf_to_pydantic.protos import p2p_validate_pb2  # isort:skip
>   File "/usr/local/lib/python3.9/dist-packages/protobuf_to_pydantic/protos/p2p_validate_pb2.py", line 31, in <module>
>     ignored = _descriptor.FieldDescriptor(
>   File "/usr/local/lib/python3.9/dist-packages/google/protobuf/descriptor.py", line 560, in __new__
>     _message.Message._CheckCalledFromGeneratedFile()
> TypeError: Descriptors cannot not be created directly.
> If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
> If you cannot immediately regenerate your protos, some other possible workarounds are:
>  1. Downgrade the protobuf package to 3.20.x or lower.
>  2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
> 
> More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
> --protobuf-to-pydantic_out: protoc-gen-protobuf-to-pydantic: Plugin failed with status code 1.
>

**Comment by so1n** [comment]
> protobuf_to_pydantic/protobuf_to_pydantic/protos/

Thanks for your feedback, I realized it was a problem with different protobuf versions reading files `protobuf_to_pydantic/protobuf_to_pydantic/protos/`

**Comment by so1n** [comment]
This issue has been fixed and merged into Master, and will be merged into version 0.1.7 later @pppwaw @caniko
